app/src/main/java/com/example/movies/FragmentMoviesList.kt

```diff
@@ -0,0 +1,24 @@
+package com.example.movies
+
+import android.os.Bundle
+import android.view.View
+import androidx.fragment.app.Fragment
+
+class FragmentMoviesList : Fragment(R.layout.fragment_movies_list) {
```

Comment:
and now apply

https://github.com/terrakok/Cicerone
for fragments switching

https://developer.android.com/topic/libraries/view-binding
for accessing views instead of findViewById